### PR TITLE
Make non-commtrack location types administrative

### PR DESCRIPTION
**Pull request by esoergel**
@benrudolph
http://manage.dimagi.com/default.asp?220904

### CODE REVIEW

File: corehq/apps/locations/management/commands/migrate_admin_status.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        locations=num_locations,
+    )
+
+
+def run_migration():
+    for location_type in with_progress_bar(get_affected_location_types()):
+        if not location_type.administrative:
+            location_type.administrative = True
+            location_type.save()
+            sleep(1)
```
Comment (benrudolph) [inline corehq/apps/locations/management/commands/migrate_admin_status.py]: does postgres fall over when you do this too fast?

File: corehq/apps/locations/management/commands/migrate_admin_status.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# One-off migration from 2016-04-04
+from optparse import make_option
+from time import sleep
+from django.core.management.base import BaseCommand
+from corehq.apps.locations.models import LocationType, SQLLocation
+from corehq.apps.es import DomainES
+from corehq.util.log import with_progress_bar
```
Comment (benrudolph) [inline corehq/apps/locations/management/commands/migrate_admin_status.py]: nifty

**Comment by benrudolph** [comment]
LGTM, assuming no tests, merging

File: corehq/apps/locations/management/commands/migrate_admin_status.py (hunk 1 above)
Comment (esoergel) [inline corehq/apps/locations/management/commands/migrate_admin_status.py]: It's probably fine, but each of these saves kicks off a celery task which does a bunch stuff, so I thought it'd be prudent to throttle that a bit.

I'll see the exact numbers when I run this, but I'm expecting < 1000 location types, so at 1 second each, that's around 15 minutes, which is totally doable.

**Comment by czue** [comment]
nice thanks for taking this on

File: corehq/apps/locations/management/commands/migrate_admin_status.py (hunk 1 above)
Comment (esoergel) [inline corehq/apps/locations/management/commands/migrate_admin_status.py]: Oh awesome, this is much less prevalent that I'd expected
> There are 24 domains, 70 loc types, and 1100 locations affected

The task finished without error

**Pull request merged**